.github/workflows/tests.yml

```
@@ -0,0 +1,18 @@
+name: Brand UI tests
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Test
+        run: yarn test
```

so you can have one step less and still use as CI

```suggestion
name: Brand UI tests

on:
  pull_request:
    branches:
    - master

jobs:
  test:

    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v1
    - name: Use Node.js 12.x
      uses: actions/setup-node@v1
      with:
        node-version: 12.x
    - name: Run tests
      run: |
        yarn
        yarn test --passWithNoTests
      env:
        CI: true
```